coreos-4/policies/SI-System_and_Information_Integrity/component.yaml

```diff
@@ -134,13 +134,13 @@
 - control_key: SI-2 (6)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response for SI-2(6) is planned. Engineering progress can be
-        tracked via:
-
-        https://issues.redhat.com/browse/CMP-528
+        The customer is responsible for ensuring that after installing an updated version
+        of software and firmware that the old version is removed. Configuration of CoreOS
+        post-install is done through OpenShift Container Platform, including software installation
+        and removal with `yum`.
```

Comment:
Actually, SI-2(6) is applicable because `yum` can remove previous versions of software. CoreOS doesn't use `yum` but the updating CoreOS should remove the previous software components. In addition, adding `fwupd` can facilitate the installing, updating, and removal of firmware.

Reply:
The text of the control enhancement states that satisfaction of the control is dependent on organizational definition of scope. This is a sticky area, and I do not feel comfortable marking this is 'partial' or 'complete' without  a justification that responsibility for satisfaction lies with CoreOS instead of the organization.

Reply:
@redhatrises and I discussed this on a call, full text of the control enhancement is "The organization removes [ Assignment: organization-defined software and firmware components ] after updated versions have been installed."
We agreed that this is an org control, and not applicable although documentation should be provided to inform the organization implementing the system.